application-task-api/src/main/java/com/xwiki/task/model/Task.java

```diff
@@ -121,7 +122,7 @@ public class Task
 
     private DocumentReference reporter;
 
-    private DocumentReference assignee;
```

Comment:
Since this is part of the public API, you shouldn't simply change the type of the property. This is a breaking change. Would be better to add a new property and deprecate the old ones.